spec/descriptions/createSyntheticCredential.md

```diff
@@ -0,0 +1,22 @@
+This API endpoint creates a credential to be used on Synthetic API Script Tests.
+
+## Mandatory Parameters:
+
+- **credentialName** The name identifying the credential. Credential names must start with a letter and can only contain letters, numbers and underscores.
+
+- **credentialValue:** Value of the credential to be used on the API Script execution. Credential values are stored encrypted and cannot be queried after creation.
+
+
+## Sample script and payload:
+- A sample script to create a simple Ping API test
```

Comment:
Since we are not using "Ping" anymore, should this read "A sample script to create an API Simple test"